### PR TITLE
Unit Test Documentation

### PR DESCRIPTION
**Pull request by jschlenger**
Additional section on unit tests

### CODE REVIEW

File: Documentation/markdown/UnitTests.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Unit Tests
+
+OIP already provides a set of unit tests to check continuously that even after changes 
+to the code, everything is still working smoothly.
+Primarily official buildingSMART IFC example files are used to check OIP for correctness
+since they come with screenshots and additional information on how a correct visualization needs to 
+look.
+In this way, specific IfcEntities can be inspected individually for correct implementation.
+
+Currently, the OIP unit tests are not automated and need to be run manually.
```
Comment (pjanck) [inline Documentation/markdown/UnitTests.md]: There should be an open issue for that - reference maybe?

File: Documentation/markdown/UnitTests.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+In order to run them, make sure you selected the unit test module in the CMake configuration options
+(for more information on the CMake configuration, take a look at the CMakeOptions).
```
Comment (pjanck) [inline Documentation/markdown/UnitTests.md]: link to cmake options?

do we really have such option?

**Review by pjanck — APPROVED** [review]
Small improvements, otherwise OK.

File: Documentation/markdown/UnitTests.md (hunk 2 above)
Comment (jschlenger) [reply Documentation/markdown/UnitTests.md]: I already had the link in there but accidentally removed it.

The BUILD_TESTS option should do that.

File: Documentation/markdown/UnitTests.md (hunk 1 above)
Comment (jschlenger) [reply Documentation/markdown/UnitTests.md]: I'm not able to find it

Comment (jschlenger) [reply Documentation/markdown/UnitTests.md]: #179

**Pull request merged**